frontend/pages/policies/ManagePoliciesPage/components/ManagePolicyAutomationsModal/ManagePolicyAutomationsModal.tsx

```diff
@@ -99,7 +99,8 @@ const ManagePolicyAutomationsModal = ({
     webhook_settings: { failing_policies_webhook: webhook },
   } = automationsConfig;
 
-  const { jira, zendesk } = availableIntegrations || {};
+  const { jira, zendesk, google_calendar: googleCalendar } =
```

Comment:
Is `google_calendar` going to be required by the update config api? Seem silly to grab it here just for the sake of sending the same thing back on submission. @getvictor?

Reply:
Awesome, thanks @getvictor